test/features/deploy_kogito_build.feature

```diff
@@ -40,42 +41,6 @@ Feature: Deploy Kogito Build
     And Local example service "<example-service>" is built by Maven with configuration:
       | native | <native> |
 
-    When Build binary <runtime> service "<example-service>" with configuration:
```

Comment:
Why did you delete this test?

Reply:
`Build <runtime> binary build with native <native> using KogitoBuild` and `Build <runtime> binary build from local example service target folder with native <native> using KogitoBuild` are performing the test AFAIU
the first one was set before the CLI could handle the upload of binaries, see https://github.com/kiegroup/kogito-operator/pull/976/files?file-filters%5B%5D=.bats&file-filters%5B%5D=.deploy&file-filters%5B%5D=.feature&file-filters%5B%5D=.go&file-filters%5B%5D=.groovy&file-filters%5B%5D=.sh&file-filters%5B%5D=No+extension#diff-aab0846fc8b6246b32aca272924400d2dd8c8306e0dc2d536a0a6c94e4162ce3L47

Reply:
ok, thanks